### DISCUSSION TITLE
Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Jul 9, 2023

### DISCUSSION THREAD

**Discussion by zorilalaurentiugit** [community/community#60675] [GitHub Education]
### Hi everyone,

Hi everyone,

I hope you're doing well! I'm excited to be part of the GitHub Education community and I'm looking forward to connecting with all of you.

I recently received an email from GitHub stating that they were unable to verify my academic affiliation for my GitHub account that I submitted on Jul 9, 2023. They mentioned that attendees of my school must verify and use their academic email address.
The thing is, I don't have an academic email address associated with my current institution. However, I'm really eager to access the educational resources and tools on GitHub. I'm currently in my fourth year of studying medicine, with two more years to go before completing my degree. Alongside my medical studies, I have a strong passion for programming, and I believe that GitHub's tools will greatly aid me in my learning process.
I wanted to ask for your advice on how I can address this issue with GitHub. Is there an alternative way to prove my student status, such as providing additional documentation? If anyone has faced a similar situation or has any suggestions, I would greatly appreciate your insights.
Thank you all for your time and assistance!
Best regards,
Elena

**Comment by samus-aran** [comment]
Hi @zorilalaurentiugit  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

You should provide evidence to support if your school has additional academic domains, has changed its academic email issuing policy, or does not provide academic emails. Reach out to your institution's IT team for this. 

Hope this helps! Let us know how you get along 👍
